core/src/main/java/io/aiven/klaw/helpers/db/rdbms/UpdateDataJdbc.java

```diff
@@ -204,6 +218,17 @@ public CRUDResponse<Topic> updateTopicRequest(TopicRequest topicRequest, String
     return saveResult;
   }
 
+  public String updateOperationalChangeRequest(
+      OperationalRequest operationalRequest, String approver) {
+    log.debug("updateOperationalChangeRequest {} {}", operationalRequest.getTopicname(), approver);
+    operationalRequest.setApprover(approver);
+    operationalRequest.setRequestStatus(RequestStatus.APPROVED.value);
+    operationalRequest.setApprovingtime(new Timestamp(System.currentTimeMillis()));
+    operationalRequestsRepo.save(operationalRequest);
+
+    return ApiResultStatus.SUCCESS.value;
+  }
```

Comment:
updateOperationalChangeRequest && declineOperationalRequest can be the same method and just accept a RequestStatus in the method signature.